### DISCUSSION TITLE
Add PR for subtitles performance

### DISCUSSION THREAD

**Discussion by Obegg** [zhongfly/mpv-build#77] [General]
There's one PR I would love to see merged and it has been delayed for super long time,
this PR is from libass and improves performance for rendering subtitles,
Here's the issue: https://github.com/mpv-player/mpv/issues/11511
Here's the PR: https://github.com/libass/libass/pull/636

**Comment by zhongfly** [comment]
I would only add extra patches to fix the compilation, not add such functional patches.
If you really want it, you can fork repo and modify.

**Reply by zhongfly** [reply]
Also, I should remind you that this pr can't be merged directly into the master branch of libass at the moment, because "This branch has conflicts that must be resolved".You'll also need to resolve conflicts if you want to merge it
